libs/data-mapper/src/lib/components/inputTypes/InputDropdown.tsx

```diff
@@ -283,14 +286,32 @@ export const InputDropdown = (props: InputDropdownProps) => {
             }
           } else {
             let someTypesMatched = false;
+            let possibleConversionFunctions = '';
             currentNode.inputs[isUnboundedInput ? 0 : inputIndex].allowedTypes.forEach((type) => {
+              const conversion = functions.find((func) => func.category === FunctionCategory.Conversion && func.outputValueType === type);
```

Comment:
Maybe this would work, maybe it would. Could have a 2 tiered message: where we suggest a function where the input matches the incoming type and the output matches the destination and if we don't find a conversion that matches the input, do what you have here.

Example: Decimal -> String
Search the conversion functions for a function that takes in a decimal as an accepted type and outputs string.

That way we can cut down on the number of results we return. Thoughts?

Reply:
So currently none of the conversion functions from the backend actually accept a specific type as input, only 'any', so we would have to get the backend on board with this.

Reply:
So we are waiting to discuss further with the dev and PM teams on this, will merge until we discuss it